en/docs/manage-message-brokers/billing-for-platform-services.md

```diff
@@ -0,0 +1,8 @@
+# Billing for Platform Services (Kafka Services)
```

Comment:
Seems like the this info is common to both databases and message brokers so better to avoid repeating and have the content in way we can point off to from both databases and brokers.